go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/vaikas/sigstore-scaffolding
+module github.com/sigstore/scaffolding
 
 go 1.16
```

Comment:
why not move straight to 1.17?

Reply:
I don't think I use anything that would require go 1.17 as the min version. But I can change if you think it makes sense?

Reply:
this project is not meant to be consumable as a lib right?

Reply:
Yeah. So I changed it :)